runtime/gc_ctrl.c

```diff
@@ -171,6 +171,8 @@ CAMLprim value caml_gc_set(value v)
                      ARCH_INTNAT_PRINTF_FORMAT "u%%\n", caml_percent_free);
   }
 
+  atomic_store_relaxed(&caml_verb_gc, Long_val (Field (v, 3)));
```

Comment:
Nitpick: in this function it is not easy to check that we are in fact taking all the parameters into account. I think that one change that would improve readability in this respect is to make all the `Long_val (Field (v, n))` read at the beginning of the function, giving names to the results, in the same order as the fields are stored in the `caml_gc_get` function. Then it would be very easy to check that they both use the same fields (0, 2, 3, 5, 8, 9, 10).

Reply:
This change looks like it could be done in a separate PR, isn't it?